terraform/organizations-accounts-cica.tf

```diff
@@ -45,10 +45,11 @@ resource "aws_organizations_account" "cica-development" {
 }
 
 resource "aws_organizations_account" "cica-test-verify" {
-  name      = "CICA Test & Verify"
+  name      = "CICA Production"
```

Comment:
Can you confirm you've followed the [How do I change the name on my AWS account?](https://aws.amazon.com/premiumsupport/knowledge-center/change-organizations-name/) guide for this - you'll need to login to the AWS root user for this AWS account.

Reply:
Have forwarded info to team

Reply:
You'll need to remove the name change in this PR until the above is completed.